bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -222,22 +311,39 @@ private boolean selectionIsGuideTarget() {
      */
     private boolean selectionIsUserTarget() {
         final TargetEnvironment env = getDataObject().getTargetEnvironment();
-        return selectedTarget().exists(t -> env.isUserPosition(t));
+        return selectedTarget().exists(env::isUserPosition);
     }
 
     /**
-     * Auxiliary method to set the selection to the specified target.
+     * Auxiliary method to determine if the current selection is coordinates.
      */
-    private void setSelectionToTarget(final SPTarget t) {
-        _curSelection = ImOption.apply(t).map(Left::new);
+    private boolean selectionIsCoordinates() {
+        return selectedCoordinates().isDefined();
     }
 
     /**
-     * Auxiliary method to set the selection to the specified group.
+     * Auxiliary method to determine if the base coordinates are explicitly defined.
      */
+    private boolean baseCoordinatesAreDefined() {
```

Comment:
This seems easy to misunderstand ... `isGhostAsterismWithExplicitlyDefinedBaseCoordinates`?

Reply:
Agree. That's verbose, but much clearer.